src/managers/views/iframe.js

```diff
@@ -6,7 +6,16 @@ import { EVENTS } from "../../utils/constants";
 import { Pane, Highlight, Underline } from "marks-pane";
 
 class IframeView {
+	/**
+	 * One section only have one View,
+	 * we can put them into the map to destroy the old one.
+	 */
+  static ViewMap = new Map();
 	constructor(section, options) {
+    const oldView = IframeView.ViewMap.get(section.href);
+    if (oldView) oldView.destroy();
+    IframeView.ViewMap.set(section.href, this);
```

Comment:
Ensure that the previous instance of the current page is destroyed to prevent promise blocking.